content/en/synthetics/private_locations/_index.md

```diff
@@ -86,15 +88,21 @@ To pull test configurations and push test results, the private location worker n
 
 {{< /site-region >}}
 
+### Permissions
+
+By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Users with the Datadog Admin and Datadog Standard roles can view private locations, search for private locations, and assign Synthetic tests to private locations. 
```

Comment:
```suggestion
By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Users with the Datadog Admin and Datadog Standard roles can view private locations, search for private locations, and assign Synthetic tests to private locations. You can get access to the **Private Locations** page by having your user upgraded to one of these two [default roles][link]. 
```

Reply:
Oh and we should also add the right links on:
- admin role
- standard role
- default roles